apps/prairielearn/src/lib/editors.ts

```diff
@@ -1054,6 +1054,9 @@ export class QuestionCopyEditor extends Editor {
     debug(`Write info.json with new title and uuid`);
     infoJson.title = names.longName;
     infoJson.uuid = this.uuid;
+    delete infoJson['sharingSets'];
+    delete infoJson['publiclyShared'];
+    delete infoJson['publiclySharedWithSource'];
```

Comment:
I was on the fence about deleting sharing info when copying a question within a course, but the fact that sharing can't be undone at this point makes me think we should not share by default when copying.

Reply:
done